src/many/src/server/module/_4_events/info.rs

```diff
@@ -1,11 +1,11 @@
 use crate::server::module::EmptyArg;
 use minicbor::{Decode, Encode};
 
-pub type TransactionsArgs = EmptyArg;
+pub type InfoArgs = EmptyArg;
 
 #[derive(Decode, Encode)]
 #[cbor(map)]
-pub struct TransactionsReturns {
+pub struct InfoReturn {
```

Comment:
`#[n(1)]` is missing. Should be 

```
; Supported list of event types.
1 => [ * event-type ],
```

according to spec in https://github.com/many-protocol/specification/pull/42